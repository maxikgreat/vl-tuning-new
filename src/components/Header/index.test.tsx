import {render} from '@testing-library/react';

import {Header} from './index';

describe('Header component', () => {
	it('should render correctly and match snapshot', () => {
		const {asFragment, baseElement} = render(<Header />);

		expect(baseElement).toBeInTheDocument();
		expect(asFragment).toMatchSnapshot();
	});
});
