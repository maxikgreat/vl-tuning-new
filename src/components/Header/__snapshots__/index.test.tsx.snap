// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render correctly and match snapshot 1`] = `
<DocumentFragment>
  <header
    class="flex justify-between items-center p-4"
  >
    <div
      class="hidden sm:block"
    >
      <h3
        class="text-base sm:text-xl mb-3"
      >
        Украина | Харьков
      </h3>
      <h3
        class="text-base sm:text-xl"
      >
        Торговый центр "Лоск"
      </h3>
    </div>
    <div
      class="w-24 sm:w-28 md:w-32 lg:w-36"
    >
      <img
        alt="company logo"
        placeholder="empty"
        src="placeholder-file"
      />
    </div>
    <div
      class="text-base sm:text-xl"
    >
      <h3
        class="mb-0 sm:mb-3"
      >
        <a
          href="tel:+380972713473"
          rel="nofollow noopener noreferrer"
        >
          +380 97 271 34 73
        </a>
      </h3>
      <h3
        class="mb-3 sm:mb-0"
      >
        <a
          href="mailto:vltuning@gmail.com"
          rel="nofollow noopener noreferrer"
        >
          vltuning@gmail.com
        </a>
      </h3>
      <div
        class="block sm:hidden"
      >
        <h3
          class="text-base sm:text-xl mb-0"
        >
          Украина | Харьков
        </h3>
        <h3
          class="text-base sm:text-xl"
        >
          Торговый центр "Лоск"
        </h3>
      </div>
    </div>
  </header>
</DocumentFragment>
`;
